docsite/docs/developer/develop-locally.md

```diff
@@ -1,16 +1,76 @@
 # Develop ob-operator locally
 
-Tips: In this tutorial, we'll disable webhook validation and run manager controller on our laptop. The controller manager will communicate with kubernetes cluster by local .kube/config configuration file.
-
-## Background
+## Introduction
 
 ob-operator depends on [kubebuilder](https://kubebuilder.io/introduction), an operator framework maintained by kubernetes SIGS. It offers convenient utilities to bootstrap an operator and manage API types in it. Like other operator frameworks, kubebuilder depends on kubernetes [controller runtime](https://github.com/kubernetes-sigs/controller-runtime) either, which is an excellent reference to know how kubernetes dispatch events and reconcile resources.
 
-## Disable Webhook and CertManager
+## Requirements
+
+- Go 1.22 or above is required to build ob-operator, you can refer to the [official website](https://go.dev/) to setup go environment.
+- [make](https://www.gnu.org/software/make/) is used for a variety of build and test actions.
+- kubebuilder is used as k8s operator framework. it's highly recommended to read [kubebuilder books](https://book.kubebuilder.io/). 
+- Access to a kubernetes cluster is required to develop and test the operator. You can use [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or [k3s](https://k3s.io/) to create a local kubernetes cluster.
+
+## Modify and Apply CRDs
+
+### Modify definition of CRDs
+
+`ob-operator` uses kubebuilder as operator framework, which generates CRD definition in YAML format from Go code lies in `api/v1alpha1` directory. If you have modified the Go code, for example adding a new field to `OBClusterSpec` in `api/v1alpha1/obcluster_types.go`, you need to regenerate the CRD definition by running `make generate manifests fmt`. Then you will see changes in `config/crd/bases/oceanbase.oceanbase.com_xxx.yaml` files.
```

Comment:
CRD is short for Custom Resource Definition, so I think the definition after it can be dropped

Reply:
Actually what the yaml describe is CRD, so the content of yaml should be definition of definition. :) Well, I'll remove it soon.